app/helpers/katello/hosts_and_hostgroups_helper.rb

```diff
@@ -91,26 +91,48 @@ def fetch_content_view(host, options = {})
       selected_content_view if selected_content_view.present?
     end
 
+    def accessible_lifecycle_environments(org, host)
+      selected = lifecycle_environment(host)
+      envs = org.kt_environments.readable
+      envs |= [selected] if selected.present? && org == selected.organization
+      envs
+    end
+
+    def accessible_content_proxies(obj)
```

Comment:
What is `obj` here? Seems very generic, but the code below makes me think this method expects a smart_proxy.